sdk/helper/keysutil/policy.go

```diff
@@ -241,6 +241,9 @@ type KeyEntry struct {
 	// This is deprecated (but still filled) in favor of the value above which
 	// is more precise
 	DeprecatedCreationTime int64 `json:"creation_time"`
+
+	// Key entry certificate chain. If set, leaf certificate key matches the keyEntry 'key'
```

Comment:
Let's add a comment that for each certificate, the first element is the leaf perhaps?